db/seeds.rb

```diff
@@ -69,4 +69,12 @@
   end
 end
 
+# Claims::User for DFE SIGN IN, we don't have an invite process yet
+Claims::SupportUser.find_or_create_by!(
+  first_name: "Catalin",
```

Comment:
Are you sure you want to use your own name here?

Reply:
Perhaps these should be environment variables?

Reply:
This has been resolved here https://github.com/DFE-Digital/itt-mentor-services/pull/184/commits/aa850c5ca2aa452233102b5b30dcd7ac478329fd

We had a chat and I think we want to allow any user with a valid dfe email or a dfe uid to login, so this means that  a support user can create a user in our app and if it has the same email address as the the dfe sign in email address we allow them to log in.

I'll change the description of the PR as well